tauri/src/app/runner.rs

```diff
@@ -200,3 +200,80 @@ fn build_webview(
       .build()?,
   )
 }
+
+#[cfg(test)]
+mod test {
+  use proptest::prelude::*;
+  use std::{fs::read_to_string, path::Path};
+  use web_view::Content;
+
+  fn init_config() -> crate::config::Config {
+    crate::config::get().expect("unable to setup default config")
+  }
+
+  #[test]
+  fn check_setup_content() {
+    let config = init_config();
+    let c = config.clone();
+
+    if cfg!(not(any(feature = "embedded-server", feature = "no-server"))) {
```

Comment:
this block could be the final "else"

Reply:
I may just remove the entire cfg! logic.  There is a way to shrink it considering the main difference is the check afterwards.